app/src/components/map/MapContainer.tsx

```diff
@@ -459,19 +146,27 @@ const MapContainer: React.FC<IMapContainerProps> = (props) => {
       maxZoom={17}
       fullscreenControl={true}
       scrollWheelZoom={scrollWheelZoom || false}>
-      <FullScreenEventHandler bounds={bounds} />
+      <FullScreenScrollingEventHandler bounds={bounds} scrollWheelZoom={Boolean(scrollWheelZoom)} />
+
+      <SetMapBounds bounds={bounds} />
+      <GetMapBounds onChange={(newBounds, newZoom) => props.onBoundsChange?.(newBounds, newZoom)} />
+
+      {drawControls && (
+        <FeatureGroup data-id="draw-control-feature-group" key="draw-control-feature-group">
+          <DrawControls
+            {...props.drawControls}
+            options={{
+              ...props.drawControls?.options,
+              // Always disable circlemarker
+              draw: { ...props.drawControls?.options?.draw, circle: false, circlemarker: false }
```

Comment:
Are we using line control as well? Previously we are not, should we disable here as well?